config/routes.rb

```diff
@@ -393,6 +393,13 @@
         patch '/cancel/:id' => 'referees#cancel', as: :cancel_referee
       end
 
+      scope '/decoupled-referees' do
```

Comment:
I like the prefixing of controllers and path helpers for clarity while this features is in-progress, but maybe the scope could just be `/references`? That way no visible change to any clients of this code when we eventually come back and remove the feature flag.

Reply:
And also this. I'll make the changes now